sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2299,6 +2291,13 @@ object GpuOverrides {
         override def convertToGpu(child: Expression): GpuExpression =
           GpuMakeDecimal(child, a.precision, a.scale, a.nullOnOverflow)
       }),
+    expr[Explode](
+      "Given an input array produces a sequence of rows for each value in the array",
+      ExprChecks.unaryProject(TypeSig.all, TypeSig.all,
+        TypeSig.ARRAY.nested(TypeSig.all), TypeSig.ARRAY.nested(TypeSig.all)),
```

Comment:
Spark supports maps, but we do not.  That should be captured here.
Spark supports user defined types as part of the nesting, we do not.  That must be removed from our list of supported input types.